src/Orchard.Azure.Tests/App.config

```diff
@@ -29,7 +29,7 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-12.0.0.0" newVersion="12.0.0.0" />
+        <bindingRedirect oldVersion="0.0.0.0-13.0.0.0" newVersion="13.0.0.0" />
```

Comment:
should it be 13.1 ?

Reply:
13.0.0.0 is the assembly version
13.0.1 is the nuget version.
So 13.0.0.0 is the good one for the binding redirect.

Reply:
Good to go then